generators/app/index.js

```diff
@@ -105,6 +105,20 @@ module.exports = class extends BaseGenerator {
             defaults: 'jhi'
         });
 
+        // This adds support for a `--domain-suffix` flag
+        this.option('domain-suffix', {
```

Comment:
"domain suffix" does not really talk to me, if you google these terms you'll find DNS related content and this is what also what it means to me. Based on description I would rather call it "entity suffix". What do others think?

Reply:
Yes, 'entity suffix' sounds better to me also.

Reply:
I called it 'domain' to match the package where the class stands. But I'm ok with 'Entity Suffix', it makes more sense.

I'll change this tomorrow.